### PR TITLE
Os/feature deleting votes and vote items

### PR DESCRIPTION
**Pull request by saienkoo**

### CODE REVIEW

File: resources/assets/templates/vote/voteDetail.hbs [hunk 1]
```diff
@@ -7,7 +7,8 @@
 
         </div>
     </div>
-    <div class="vote-body">
+    <div class="vote-body" style="position: relative;">
```
Comment (binaryvo) [inline resources/assets/templates/vote/voteDetail.hbs]: move style to appropriate css file

File: resources/assets/templates/vote/voteHeader.hbs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <!--<div class="vote-head">-->
-    <div class="vote-header">
+    <div class="vote-header" style="position: relative;">
```
Comment (binaryvo) [inline resources/assets/templates/vote/voteHeader.hbs]: move style to appropriate css file

File: resources/assets/templates/vote/voteItem.hbs [hunk 3]
```diff
@@ -4,7 +4,7 @@
             <img src="/images/user.png">
         </div>
     </div>
-    <div class="votes-item-body">
+    <div class="votes-item-body" style="position: relative;">
```
Comment (binaryvo) [inline resources/assets/templates/vote/voteItem.hbs]: move style to appropriate css file

File: resources/assets/js/app/views/votes/VoteAnswersItem.js [hunk 4]
```diff
@@ -2,10 +2,22 @@ var Marionette = require('backbone.marionette');
...
+    ui: {
+        deleteButton: '.delete-button'
+    },
+    events: {
+        'click @ui.deleteButton': function () {
+            this.model.destroy();
+        }
+    },
     serializeData: function () {
+        var tempmeta = this.model.getMeta()[this.model.get('id')];
```
Comment (binaryvo) [inline resources/assets/js/app/views/votes/VoteAnswersItem.js]: Probably it would be better create method getMetaById()?

Comment (TatsuUkraine) [reply resources/assets/js/app/views/votes/VoteAnswersItem.js]: Meta can contain different data, including object

**Pull request closed without merging**